### ISSUE TITLE
Warlock Drain soul not working consistently

### ISSUE THREAD

**Issue by TalyTea** [gamefreedomgit/Frostmourne#705]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (https://www.wowhead.com/wotlk)

**Description:**
Playing Affliction warlock after dotting up and starting to use drain soul my hands will glow purple like the drain soul fx but itll say interupted. It may do it once may do it twice but nothing is interupting me. Its only in dungeons so far.

**How to reproduce:**
after playing lock for awhile and dungeon grinding ive come to notice its whenever another warlock is in the party. As often times when im the only warlock it never happens. 

**How it should work:**
I should be able to channel drain soul not be interupted all the time before i even start it.

**Database links:**

**Comment by Robosterone** [comment]
You're most likely missing the cast :)

**Comment by TalyTea** [comment]
I dont think so since its well beyond the gcd i pay attention to the gcd and itll do it multiple times before finally working a good few seconds after gcd ended.

**Comment by TalyTea** [comment]
Can channels miss? I actually dont even remember missing a channel its been awhile. I thought channels start 100% of the time but you can miss over the course of the channel.

**Comment by Godmatik** [comment]
Yes it can miss same as any other spell

**Comment by WhitemaneIsa** [comment]
As there has been no further info/complaints, this will be closed for now.
